eq/deflect/eventHandler.cpp

```diff
@@ -227,18 +233,20 @@ void EventHandler::_processEvents( const Proxy* proxy )
             event.pointerMotion.dx = -deflectEvent.dx;
             event.pointerMotion.dy = -deflectEvent.dy;
             break;
-        case ::deflect::Event::EVT_TAP_AND_HOLD:
+        case ::deflect::Event::EVT_PINCH:
         {
-            const Proxy::NavigationMode mode =
-                    _proxy->getNavigationMode() == Proxy::MODE_PAN
-                        ? Proxy::MODE_ROTATE : Proxy::MODE_PAN;
-            _proxy->setNavigationMode( mode );
-            Event windowEvent;
-            windowEvent.originator = window->getID();
-            windowEvent.serial = window->getSerial();
-            windowEvent.type = Event::WINDOW_EXPOSE;
-            window->processEvent( windowEvent );
-        } break;
+            event.type = Event::CHANNEL_POINTER_WHEEL;
+            event.pointerWheel.x = x;
+            event.pointerWheel.y = pvp.h - y;
+            event.pointerWheel.buttons = PTR_BUTTON_NONE;
+            const auto dx = deflectEvent.dx * pvp.w;
+            const auto dy = deflectEvent.dy * pvp.h;
+            const auto sign = dx + dy;
+            const auto zoom = std::copysign( std::sqrt( dx*dx + dy*dy ), sign );
+            event.pointerWheel.xAxis = 0.f;
+            event.pointerWheel.yAxis = zoom / 40.f;
```

Comment:
It's a bit of magic that was copied from the wheel event above and I cannot say that I understand what it corresponds to. Documentation for xAxis states: 'wheel rotation in clicks'. That value is then multiplied again by 0.05f in eqPly / seq / livre to move the camera...